sensors/runfolder_sensor.yaml

```diff
@@ -2,7 +2,7 @@
   class_name: "RunfolderSensor"
   entry_point: "runfolder_sensor.py"
   description: "Sensor which monitors runfolders"
-  poll_interval: 15
+  poll_interval: 36000
```

Comment:
This is just to not make it fill up your machine with unsuccessful jobs, since right now it's not working properly...